app/js/rest/encounterRest.js

```diff
@@ -0,0 +1,35 @@
+
+import axiosInstance from '../config';
+
+const api = {
+  fetchEncountersByPatient: (patient, encounterType, provider, location) => axiosInstance.get(`encounter/?patient=${patient}&encounterType=${encounterType}&provider=${provider}&location=${location}&v=full`)
```

Comment:
"Full" represetations are pretty huge... if we can fetch a custom rep that meets our needs closely, that would be better. Don't we really just need the encounterDatetime of the encounters?